authors/authors.json

```diff
@@ -48,6 +64,14 @@
         "sympy_commits": 186
     },
     {
+        "email": "sean.v.775@gmail.com",
+        "github_id": "flacjacket",
+        "institution": "??",
+        "institution_address": "??",
```

Comment:
University of Illinois at Urbana-Champaign
1110 W Green St, Urbana, IL 61801, USA